### PR TITLE
Adding more information about the app

### PR DESCRIPTION
**Pull request by czearing**
#### Description of changes

Adding more information about how the app works. Also cleaning up some bugs in the contact section and global styling.

### CODE REVIEW

File: src/components/contactForm.tsx [hunk 1]
```diff
@@ -82,41 +92,44 @@ export const ContactForm = () => {
...
-              {state.succeeded ? 'Thanks!' : state.submitting ? 'Sending...' : 'Submit'}
-            </Button>
-          </Stack>
-          <div style={{ opacity: !emailError ? 0 : 1 }}>
-            <Body color="danger" id={helperTextId}>
-              Please enter a valid email
-            </Body>
-          </div>
-        </form>
+        <>
```
Comment (calebjohn24) [inline src/components/contactForm.tsx:95]: I like the icons, I would remove the (optional) and keep the or, its easier to understand.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -100,13 +102,37 @@ const Home: NextPage = () => {
...
       <Divider />
-      {/* <Container appearance="relaxed" horizontalAlignment="center" verticalAlignment="center">
+      <Container appearance="relaxed" horizontalAlignment="center" verticalAlignment="center">
+        <Stack appearance="relaxed" horizontalAlignment="center" verticalAlignment="center">
+          <Card>
+            <Image src="/image/store.png" width="208px" height="453px" />
+          </Card>
+          <Stack vertical>
+            <Title align="center">Build a store in 5 minutes</Title>
+            <Title align="center">on your phone</Title>
```
Comment (calebjohn24) [inline src/pages/index.tsx]: Build a storefront in 5 minutes, all from your phone.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -100,13 +102,37 @@ const Home: NextPage = () => {
...
+          </Stack>
+        </Stack>
+      </Container>
+      <Divider />
+
+      <Container appearance="relaxed" horizontalAlignment="center" verticalAlignment="center">
         <Stack appearance="relaxed" horizontalAlignment="center" verticalAlignment="center">
-          <Title align="center">Start selling in 5 minutes.</Title>
-          <Card>Content Here</Card>
+          <Title align="center">Easily manage payments</Title>
```
Comment (calebjohn24) [inline src/pages/index.tsx]: Easily process payments

File: src/pages/index.tsx [hunk 4]
```diff
@@ -100,13 +102,37 @@ const Home: NextPage = () => {
...
         </Stack>
       </Container>
-      <Divider /> */}
+      <Divider />
+      <Container appearance="relaxed" horizontalAlignment="center" verticalAlignment="center">
+        <Stack appearance="relaxed" horizontalAlignment="center" verticalAlignment="center">
+          <Card>
+            <Image src="/image/orders.png" width="208px" height="453px" />
+          </Card>
+          <Title align="center">View orders</Title>
```
Comment (calebjohn24) [inline src/pages/index.tsx]: Manage and ship orders without leaving the app

File: src/pages/index.tsx [hunk 5]
```diff
@@ -142,6 +168,7 @@ const Home: NextPage = () => {
       >
         <Stack vertical horizontalAlignment="center">
           <Header1 align="center">Join the waitlist (limited space)</Header1>
+          <Body>Please provide at least one method of contact.</Body>
```
Comment (calebjohn24) [inline src/pages/index.tsx]: revert, this sounds mechanical

File: src/components/contactForm.tsx (hunk 1 above)
Comment (czearing) [reply src/components/contactForm.tsx:95]: It ends up looking strange on mobile when it collapses when using "or". Having "optional" in the label is a fix for this.

Comment (czearing) [reply src/components/contactForm.tsx:95]: For some more context: 
image
image

Comment (calebjohn24) [reply src/components/contactForm.tsx:95]: Pick one

File: src/pages/index.tsx [hunk 6]
```diff
@@ -100,13 +105,60 @@ const Home: NextPage = () => {
...
+      </Container>
+      <Divider />
+      <Container appearance="relaxed" horizontalAlignment="center" verticalAlignment="center">
         <Stack appearance="relaxed" horizontalAlignment="center" verticalAlignment="center">
-          <Title align="center">Start selling in 5 minutes.</Title>
-          <Card>Content Here</Card>
+          <Stack vertical>
+            <Title className={styles.textMaxWidth}>Easily process payments</Title>
+            <Body className={styles.textMaxWidth} size={500}>
+              Process and analyze your customers payments using our app. We also provide analytical tools to track your
```
Comment (calebjohn24) [inline src/pages/index.tsx:126]: Easily process payments, and manage relationships with customers.

**Pull request merged**